### PR TITLE
Initial draft

### PR DESCRIPTION
**Pull request by bobmshannon**
### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/agent/guide/private-link.md [hunk 1]
```diff
@@ -231,6 +231,245 @@ The VPCs with Private Hosted Zone (PHZ) attached need to have a couple of settin
...
+
+8. Click on the VPC endpoint ID to check its status.
+9. Wait for the status to move from _Pending_ to _Available_. This can take up to 10 minutes.
+10. After creating the endpoint, use VPC peering to make the PrivateLink endpoint available in another region to send telemetry to Datadog over PrivateLink. For more information, read the [Work With VPC Peering connections][2] page in AWS.
+
+{{< img src="agent/guide/private_link/vpc_status.png" alt="VPC status" style="width:80%;" >}}
+
+### Amazon Route53
+
+1. Create a [Route53 private hosted zone][3] for each service you have created an AWS PrivateLink endpoint for. Attach the private hosted zone to the VPC in `us-east-1`.
```
Comment (brett0000FF) [inline content/en/agent/guide/private-link.md:378]:
```suggestion
1. Create a [Route53 private hosted zone][3] for each service you have created an AWS PrivateLink endpoint for. Attach the private hosted zone to the VPC in `ap-northeast-1`.
```

File: content/en/agent/guide/private-link.md [hunk 2]
```diff
@@ -231,6 +231,245 @@ The VPCs with Private Hosted Zone (PHZ) attached need to have a couple of settin
...
+
+{{< img src="agent/guide/private_link/create-an-a-record.png" alt="Create an A record" style="width:90%;" >}}
+
+3. Configure VPC peering and routing between the VPC in `ap-northeast-1` that contains the Datadog PrivateLink endpoints and the VPC in the region where the Datadog Agents run.
+
+4. If the VPCs are in different AWS accounts, the VPC containing the Datadog Agent must be authorized to associate with the Route53 private hosted zones before continuing. Create a [VPC association authorization][5] for each Route53 private hosted zone using the region and VPC ID of the VPC where the Datadog Agent runs. This option is not available in the AWS Console. It must be configured using the AWS CLI, SDK, or API.
+
+5. Edit the Route53 hosted zone to add the non-us-east-1 VPC.
+
+{{< img src="agent/guide/private_link/edit-route53-hosted-zone.png" alt="Edit a Route53 private hosted zone" style="width:80%;" >}}
```
Comment (brett0000FF) [inline content/en/agent/guide/private-link.md:419]: Similar to the previous comment, we could update this screenshot to show AP1, but it's also not a big deal (and might be a headache to to change this for future supported regions anyways).

**Review by brett0000FF — APPROVED** [review]
~Looks good! Thanks for adding this content.~

~It looked like there were a few places we didn't swap us1 with ap1, so I added those as suggestions.~

~Once you address those, let me know when you'd like to merge this!~

Edit: I went ahead and took a stab at making this page dynamic based on the selected region. I think this is better than duplicating so much content. Let me know what you think @bobmshannon .

**Comment by bobmshannon** [comment]
Thank you! This looks good. We do need to make sure the entry for each "PrivateLink service name" is generated dynamically with the right value. Right now with the most recent changes that is not the case. But I may be able to make that work using your short code example 👍

**Comment by brett0000FF** [comment]
> Thank you! This looks good. We do need to make sure the entry for each "PrivateLink service name" is generated dynamically with the right value. Right now with the most recent changes that is not the case. The "PrivateLink service name" contains not only the region but a unique ID that differs between which Datadog site they are applicable for. But I may be able to make that happen using your short code example 👍

Great! Sounds good. I removed the extra regions for now, but feel free to play around with this. Thanks a bunch!

**Comment by bobmshannon** [comment]
@brett0000FF I made some changes based on my initial comment. Would you mind taking another look at this and providing any feedback for improvement? This latest revision should contain everything we'd need to document the PrivateLink feature for AP1.

**Review by brett0000FF — APPROVED** [review]
Updated this to take advantage of the existing  region-param shortcode, which references values in regions.config.js.

@bobmshannon - I compared the output in my local changes with your staging site, and it looked identical. Let me know if I missed anything. Otherwise, I think we are good! Thanks for your patience.

**Comment by bobmshannon** [comment]
@brett0000FF Thank you, this looks great. I appreciate your assistance.

If you don't mind, I'd like to keep this PR in the draft state until we're ready to soft-launch and make the documentation live in about a month or two.

**Comment by brett0000FF** [comment]
@bobmshannon - Sounds good to me! Just let me know when you're ready to merge, and I can help out. I'll make a note to follow up.

**Pull request merged**